apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -28,7 +28,7 @@ function LaunchWpcomWelcomeTour() {
 
 	const localeSlug = useLocale();
 	// Preload first card image (others preloaded after open state confirmed)
-	new window.Image().src = getTourSteps( localeSlug )[ 0 ].meta.imgSrc;
+	new window.Image().src = usePrefetchTourAssets( [ getTourSteps( localeSlug )[ 0 ] ] );
```

Comment:
Hmmm...I think `usePrefetchTourAssets` returns `undefined` and it would cause the page to load the undefined assets.

I think we only need the following one
```js
usePrefetchTourAssets( [ getTourSteps( localeSlug )[ 0 ] ] );
```